bootspec/src/generation.rs

```diff
@@ -33,6 +33,17 @@ impl Generation {
     }
 }
 
+impl TryInto<v1::GenerationV1> for Generation {
```

Comment:
We shouldn't manually implement `{Try,}Into` ourselves since we get that for free from `{Try,}From`: https://doc.rust-lang.org/std/convert/trait.TryInto.html

> Library authors should usually not directly implement this trait, but should prefer implementing the [TryFrom](https://doc.rust-lang.org/std/convert/trait.TryFrom.html) trait, which offers greater flexibility and provides an equivalent TryInto implementation for free, thanks to a blanket implementation in the standard library. For more information on this, see the documentation for [Into](https://doc.rust-lang.org/std/convert/trait.Into.html).